docs/design/vcpu-handling.md

```diff
@@ -157,6 +157,32 @@ docker run --cpus 4 -ti debian bash -c "nproc; cat /sys/fs/cgroup/cpu,cpuacct/cp
 400000  # cfs quota
 ```
 
+## Virtual CPU handling without hotplug
+
+In some cases, the hardware and/or software architecture being utilized does not support
+hotplug. For example, Firecracker VMM does not support CPU or memory hotplug. Similarly,
+the current Linux Kernel for aarch64 does not support CPU or memory hotplug. To appropriately
+size the virtual machine for the workload within the container or pod, we provide a `static_sandbox_resource_mgmt`
```

Comment:
Why do you need a flag, if we can statically determine it based on hypervisor?
* For users of the hypervisor that need it, that means they need to enable it, so it's inconvenient
* Is there any valid reason to enable it e.g. for qemu?